src/app/desktop/components/News.js

```diff
@@ -200,38 +201,44 @@ function useInterval(callback, delay) {
 function News({ style, news }) {
   const [currentImageIndex, setCurrentImageIndex] = useState(0);
   const ContextTheme = useContext(ThemeContext);
+  const showNews = useSelector(state => state.settings.showNews);
 
   useInterval(() => {
     if (currentImageIndex < 9) {
       setCurrentImageIndex(currentImageIndex + 1);
     } else setCurrentImageIndex(0);
   }, 5000);
 
-  return news.length !== 0 ? (
-    <Carousel style={style}>
-      <SelectNews
-        news={news}
-        setCurrentImageIndex={setCurrentImageIndex}
-        currentImageIndex={currentImageIndex}
-      />
-      <ImageList news={news} currentImageIndex={currentImageIndex} />
-    </Carousel>
-  ) : (
-    <StyledContentLoader
-      primaryColor={ContextTheme.shade11}
-      secondaryColor={ContextTheme.shade10}
-    >
-      <rect
-        rx="0"
-        ry="0"
-        width="830"
-        height="158"
-        css={`
-          border-radius: 2px;
-        `}
-      />
-    </StyledContentLoader>
-  );
+  if (showNews) {
+    return news.length !== 0 ? (
+      <Carousel style={style}>
+        <SelectNews
+          news={news}
+          setCurrentImageIndex={setCurrentImageIndex}
+          currentImageIndex={currentImageIndex}
+        />
+        <ImageList news={news} currentImageIndex={currentImageIndex} />
+      </Carousel>
+    ) : (
+      <StyledContentLoader
+        primaryColor={ContextTheme.shade11}
+        secondaryColor={ContextTheme.shade10}
+      >
+        <rect
+          rx="0"
+          ry="0"
+          width="830"
+          height="158"
+          css={`
+            border-radius: 2px;
+          `}
+        />
+      </StyledContentLoader>
+    );
```

Comment:
What I would do, is add a `if(!showNews) return null` before the return of the news and don't change anything else